bcp/package.json

```diff
@@ -44,9 +44,10 @@
     "core-js": "^3.2.1",
     "date-fns": "^2.5.1",
     "font-awesome": "^4.6.3",
-    "moh-common-lib": "^3.3.5",
+    "moh-common-lib": "^3.5.0",
     "moment": "^2.24.0",
     "mygovbc-bootstrap-theme": "^0.4.0",
+    "ng-recaptcha": "^6.1.0",
```

Comment:
Is the inclusion of the "ng-recaptcha" package required to use the latest version of the common library? I thought it's already being included as a dependency when building the common lib.

Reply:
Only to use the recaptcha component... I couldn't find a way to get it to automatically handle the dependency, so if you are using the recaptcha component ng-recaptcha must be installed on the project as well.

Reply:
Gotcha. I just wanted to make sure the common library will still work for other applications (application with only standard captcha) without adding the additional dependency.

Reply:
It still builds and works with the MSP project, which is unchanged, while linked to the updated common library(without changes to it's dependencies).

Reply:
Excellent. That's what we need.